api/api/controllers/active_response_controller.py

```diff
@@ -1,7 +1,58 @@
+
 # Copyright (C) 2015-2019, Wazuh Inc.
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 
+import asyncio
+import connexion
+import logging
+
+import wazuh.active_response as active_response
+from wazuh.cluster.dapi.dapi import DistributedAPI
+from ..models.active_response_model import ActiveResponse
+from ..util import remove_nones_to_dict
+
+loop = asyncio.get_event_loop()
+logger = logging.getLogger('wazuh.active_response_controller')
+
+
+def run_command(pretty=False, wait_for_complete=False, agent_id='000'):
+    """
+    Runs an Active Response command on a specified agent
+
+    :param pretty: Show results in human-readable format
+    :type pretty: bool
+    :param wait_for_complete: Disable timeout response
+    :type wait_for_complete: bool
+    :param agent_id: Agent ID. All posible values since 000 onwards
+    :type agent_id: str
+    :param command: Command running in the agent. If this value starts by !, then it refers to a script name instead of a command name
+    :type command: str
+    :param custom: Whether the specified command is a custom command or not
+    :type custom: bool
+    :param arguments: Command arguments
+    :type arguments: str
+
+    :rtype: dict
+    """
+
+    # get body parameters
+    if connexion.request.is_json:
+        active_response_model = ActiveResponse.from_dict(connexion.request.get_json())
+    else:
+        return 'ERROR', 400
```

Comment:
This error message is not acceptable. We should tell the user why the input body is wrong and give a URL to doc.